README.md

```diff
@@ -156,6 +156,6 @@ Graphql\init(new \GraphQL\Schema(['query' => $query(), 'mutation' => $mutation()
 ---
 More action here: [siler-examples](https://github.com/leocavalcante/siler-examples)
 
-⚠️️ **This is a work in progress, API may change** 🚧<br>
-But if you give a try, I'd love the get some feedback<br>
```

Comment:
@leocavalcante Oops, sorry, it really looks bad, missed out of sight, added blank lines instead of br tag.